frontend/packages/kubevirt-plugin/integration-tests-cypress/fixtures/data-volume.json

```diff
@@ -0,0 +1,27 @@
+{
+  "apiVersion": "cdi.kubevirt.io/v1beta1",
+  "kind": "DataVolume",
+  "metadata": {
+    "name": "foo",
+    "namespace": "bar",
+    "annotations": {
+      "cdi.kubevirt.io/storage.bind.immediate.requested": "true"
+    }
+  },
+  "spec": {
+    "pvc": {
```

Comment:
It needs to add storageClassName for the PVC to accept STORAG_CLASS from environment variable.

Reply:
fixed